corehq/pillows/mappings/app_mapping.py

```diff
@@ -1,6 +1,6 @@
 from corehq.util.elastic import es_index
 
-APP_INDEX = es_index("hqapps_2016-01-05_1821")
+APP_INDEX = es_index("hqapps_2016-01-29_0013")
```

Comment:
just for my own understanding - are we resetting this just so that it rebuilds the app pillow from scratch off of the changes feed from the new db?